vms/avm/state_test.go

```diff
@@ -75,18 +74,15 @@ func TestSetsAndGets(t *testing.T) {
 
 	env.vm.state.AddUTXO(utxo)
 	env.vm.state.AddTx(tx)
-	env.vm.state.AddStatus(txID, choices.Accepted)
+	env.vm.state.DeleteStatus(txID)
```

Comment:
Can we document why we do this? It's not super obvious what we're doing here, and the test passes without this line. OTOH we're removing `DeleteStatus` soon so maybe we don't need to?

Reply:
It isn't needed - I felt it was most clear to always delete the status when accepting a tx to make it clear that the status will be reported correctly.

Reply:
I removed the `DeleteStatus` function entirely.